tests/test_field.py

```diff
@@ -25,3 +25,12 @@ def test_invalid_operations_on_fields():
     for op in operators:
         with pytest.raises(FieldsNotIdentical):
             op(field1(2), field2(3))
+
+
+def test_sqrt():
```

Comment:
use the `galois_field` fixture instead of redefining the field, look at `test_phase1` in `test_powermixing.py`.